packs/aws/requirements.txt

```diff
@@ -1 +1,2 @@
 boto
+boto3
```

Comment:
Just curious, why do we also need `boto3`?

It looks like sensor uses it. I would assume SQS is not supported in `boto`, or something else?